pkgs/by-name/hx/hxtools/package.nix

```diff
@@ -0,0 +1,43 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  pkg-config,
+  zstd,
+  libHX,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "hxtools";
+  version = "20231224";
+
+  src = fetchurl {
+    url = "https://inai.de/files/hxtools/hxtools-${finalAttrs.version}.tar.zst";
+    hash = "sha256-TyT9bsp9qqGKQsSyWCfd2lH8ULjqJ5puMTw2TgWHV5c=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    zstd
+  ];
+
+  buildInputs = [
+    libHX
+  ];
+
+  strictDeps = true;
+
+  meta = with lib; {
+    homepage = "https://inai.de/projects/hxtools/";
+    description = "Collection of small tools over the years by j.eng";
+    # Taken from https://codeberg.org/jengelh/hxtools/src/branch/master/LICENSES.txt
+    license = with licenses; [
+      mit
+      bsd2Patent
+      lgpl21Plus
+      gpl2Plus
+    ];
+    maintainers = with maintainers; [ meator ];
+    platforms = platforms.all;
+  };
```

Comment:
```suggestion
  meta = {
    homepage = "https://inai.de/projects/hxtools/";
    description = "Collection of small tools over the years by j.eng";
    # Taken from https://codeberg.org/jengelh/hxtools/src/branch/master/LICENSES.txt
    license = with lib.licenses; [
      mit
      bsd2Patent
      lgpl21Plus
      gpl2Plus
    ];
    maintainers = with lib.maintainers; [ meator ];
    platforms = lib.platforms.all;
  };
```

Reply:
Why?

Reply:
use of `with lib;` is considered as an anti-pattern. See #208242

Reply:
no worries. i also hadn't had it on my mind when reviewing and only stumbled over it in another pr.